package.json

```diff
@@ -69,16 +69,19 @@
     "postcompile": "cpy ./src/types/* ./build/src/types",
     "fix": "gts fix",
     "prepare": "npm run compile",
-    "pretest": "npm run compile",
+    "pretest": "npm run compile && node ./bin/install-winston2.js",
     "posttest": "npm run check"
   },
   "dependencies": {
     "@google-cloud/logging": "^3.0.2",
     "@sindresorhus/is": "^0.10.0",
-    "lodash.mapvalues": "^4.6.0"
+    "lodash.mapvalues": "^4.6.0",
+    "logform": "^1.9.1",
+    "semver": "^5.5.1",
+    "winston-transport": "^4.2.0"
   },
   "devDependencies": {
-    "@google-cloud/common": "^0.24.0",
+    "@google-cloud/common": "^0.23.0",
```

Comment:
Can we keep this at `24`?

Reply:
having 24 with the combination of other libraries is causing a getDefaultProjectId is deprecated error.
this error started and is only emitted when i rebased the changes in for testing error reporting.

I'm not sure if its worth spending more time.

Reply:
i think this will be fixed when a new version of @google-cloud/common-grpc is published. the published version 0.8.0 depends on @google-cloud/common 0.23.0

having 2 versions of @google-cloud/common seems to be a recipe for warnings